versions.properties

```diff
@@ -0,0 +1,79 @@
+#### Dependencies and Plugin versions with their available updates.
+#### Generated by `./gradlew refreshVersions` version 0.60.3
+####
+#### Don't manually edit or split the comments that start with four hashtags (####),
+#### they will be overwritten by refreshVersions.
+####
+#### suppress inspection "SpellCheckingInspection" for whole file
+#### suppress inspection "UnusedProperty" for whole file
+
+version.firebase-bom=32.7.0
+
+version.org.json..json=20231013
+
+# Note: updating past 11 will introduce new rules requiring some big formatting changes
+version.org.jlleitschuh.gradle..ktlint-gradle=11.6.1
+##                                # available=12.0.2
+##                                # available=12.0.3
+##                                # available=12.1.0
+
+version.org.jetbrains.dokka..dokka-gradle-plugin=1.9.10
+
+version.mockk=1.13.9
+
+version.kotlin=1.9.22
+## # available=2.0.0-Beta1
+## # available=2.0.0-Beta2
+
+version.junit.junit=4.13.2
+
+version.androidx.test.runner=1.5.2
+##               # available=1.5.3-alpha01
+##               # available=1.6.0-alpha01
+##               # available=1.6.0-alpha02
+##               # available=1.6.0-alpha03
+##               # available=1.6.0-alpha04
+##               # available=1.6.0-alpha05
```

Comment:
theres a config setting to refreshVersions that should ignore listing alpha versions like this if you want to add that

Reply:
oh yes please, will do